python/core/auto_generated/raster/qgsrasterdataprovider.sip.in

```diff
@@ -786,6 +786,8 @@ Data providers that have NativeRasterAttributeTable provider capability will try
 .. versionadded:: 3.30
 %End
 
+    virtual QString bandDescription( int bandNumber );   // Note: not const because GDAL init on demand
+
```

Comment:
Looks like sipify isn't liking the comment here. Does it work if you out an extra blank line above it?